mage_ai/frontend/pages/404.tsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from 'react';
+import { redirectToUrl } from '@utils/url';
+import Headline from '@oracle/elements/Headline';
+import Text from '@oracle/elements/Text';
+import Spacing from '@oracle/elements/Spacing';
+
+function NotFoundPage() {
+  const [secondsToRedirect, setSecondsToRedirect] = useState(5);
+  useEffect(() => {
+    const i = setInterval(() => {
+      if (secondsToRedirect > 0) {
+        setSecondsToRedirect(secondsToRedirect - 1);
+      } else {
+        redirectToUrl('/pipelines');
+      }
+    }, 1000);
+
+    return () => clearInterval(i);
+  }, [secondsToRedirect]);
+
+  return (
+    <main
+      style={{
+        alignItems: 'center',
+        display: 'flex',
+        flexDirection: 'column',
+        height: '100vh',
+        marginTop: '16rem',
+      }}
+    >
+      <Spacing mb={1}>
```

Comment:
Can you add the Mage logo here and change the Spacing margin prop to `my` with value of 2 (so it'd be 16px of padding above and below the "404 - Page Not Found")? You can import the logo with the following:
```
import GradientLogoIcon from '@oracle/icons/GradientLogo';
```